src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -39,10 +43,47 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/App.tsx** to the following:
+@diff ../../../exercises/react-native/03-creating-a-new-app/04-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx only
+
+</details>
+
+## Objective 2: Conditions and loops
+
+TODO
+
+### Concept TODO
+
+TODO
+
+### Setup 2
+
+✏️ Update **src/App.tsx** to the following:
+
+@sourceref ../../../exercises/react-native/04-intro-to-jsx/01-solution/App.tsx
```

Comment:
Same here, under Setup the diff should be between the last solution and either this problem folder (if there is one) or this exercise’s solution folder.